modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/security/DoubleSignDetectRunner.scala

```diff
@@ -0,0 +1,37 @@
+package org.tessellation.sdk.infrastructure.security
+
+import cats.effect.Async
+import cats.effect.std.Supervisor
+import cats.syntax.flatMap._
+import cats.syntax.foldable._
+
+import scala.concurrent.duration.DurationInt
+
+import org.tessellation.sdk.domain.Daemon
+import org.tessellation.sdk.domain.fork.ForkInfoStorage
+import org.tessellation.sdk.domain.security.{DoubleSignDetect, DoubleSignDetectRunner}
+
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+object DoubleSignDetectRunner {
```

Comment:
what does `Runner` suffix mean? is it a program, or a service? looks like a program

Reply:
Changed to `DoubleSignDetectProgram`.

Reply:
Please check how other programs in the repository are constructed/named

Reply:
Combined `DoubleSignDetect` and `DoubleSignDetectProgram`.